apps/website/build-scripts/create-data.ts

```diff
@@ -8,10 +8,17 @@ import { isActiveAmbassadorKey } from "@alveusgg/data/src/ambassadors/filters";
 import { camelToKebab, sentenceToKebab } from "../src/utils/string-case";
 import { typeSafeObjectKeys } from "../src/utils/helpers";
 
-mkdirSync(join(__dirname, "../src/data/generated"), { recursive: true });
+const dirname = new URL(".", import.meta.url).pathname;
+
+mkdirSync(join(dirname, "../src/data/generated").replace("\\", ""), {
```

Comment:
```suggestion
mkdirSync(new URL("../src/data/generated", import.meta.url), {
```
Instead of using a weird hack to get a dirname equivalent, can we just use the proper way to do it with URL directly (please apply the same to the other paths below)